kuma/wiki/templatetags/jinja_helpers.py

```diff
@@ -263,3 +264,32 @@ def wiki_url(path):
     if fragment:
         new_path += '#' + fragment
     return new_path
+
+
+def get_bare_tag(elem):
+    """
+    Returns the tag without the leading namespace
+    """
+    return elem.tag.rsplit('}', 1)[-1]
+
+
+@library.global_function
+def include_svg(path, title=None):
+    """
+    Load SVG from file system. If a title was passed, replace the value of
+    the current title tag with the new title. Return the SVG element
+    """
+    env = jinja2.Environment(loader=jinja2.FileSystemLoader('./jinja2'))
+    tmplString = env.get_template(path).render()
+    svgMarkup = jinja2.Markup(pq(tmplString).outerHtml())
```

Comment:
This should move to an ``else:`` branch for when the title is unset, otherwise you doing a bit of work to create ``svgMarkup`` twice, and throwing one away.